paper.md

```diff
@@ -0,0 +1,70 @@
+    equal-contrib: true
+    affiliation: 1
+  - name: Rick Lumpkin
+    orcid: 0000-0002-6690-1704
+    equal-contrib: true
+    affiliation: 3
+
+affiliations:
+ - name: Rosenstiel School of Marine, Atmospheric, and Earth science, University of Miami
+   index: 1
+ - name: Florida State University
+   index: 2
+ - name: NOAA Atlantic Oceanographic and Meteorological Laboratory
+   index: 3
+date: 10 April 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Lagrangian data in Earth sciences are unique because they do not conform to established standards related to dimensions, coordinates, and organizational structures. In addition, because they convolve spatial and temporal information, Lagrangian data necessitate specific processing and analysis tools for their scientific and operational exploitations. The Clouddrift Python library offers tools to process and analyze Lagrangian data with an emphasis on the ragged array representation.
+
+# Statement of need
+
+In Earth, Ocean, Geo-, and Atmospheric Science, Eulerian typically refers to a type of data  acquired or simulated at a particular fixed point or region in space. Eulerian data often materialize on spatiotemporal grids with monotonic coordinates (e.g. latitude, longitude, depth, time) for which popular Python tools such as [xarray](https://docs.xarray.dev/en/stable/) are naturally suited. Eulerian data are often contrasted with Lagrangian data. Such data refers to information acquired by observing platforms drifting with a flow they are embedded within, but also more broadly refers to the data originating from uncrewed platforms, vehicles, virtual particles, atmospheric phenomena such as tropical cyclones, and even animals that gather data along their natural but complicated paths.  Because such paths traverse both spatial and temporal dimensions, Lagrangian data often convolve spatial and temporal information that cannot always and readily be organized, cataloged, and stored in common data structures and file formats with the help of common libraries and standards. As an example, the concepts of dimensions and coordinates for Lagrangian data are ambiguous and not clearly established. As such, for both data generators and data users, Lagrangian data present challenges that the Clouddrift Python library aims to overcome. 
+
+# Scope and key features
+
+The scope of the Clouddrift library includes: 
+
+1. **Working with contiguous ragged array representations of data, wether they originate from geosciences or any other field**. Ragged array representations are typically useful when the data lengths of the instances of a feature (variable) are not all equal. With such representations the data for each features are stored contiguously in memory, and the number of elements that each feature has is contained in a count variable which Clouddrift calls *rowsize*. 
+![Ragged array representation for Lagrangian data.\label{fig:ragged_array}](./docs/img/ragged_array.png)       
```

Comment:
we have to include the image in the path (or include the folder).

Reply:
which path?

Reply:
you have the figure in `docs/img/ragged_array.png` here. Currently there is only `paper.md` and `paper.bib`.